chain/client/src/tests/query_client.rs

```diff
@@ -252,7 +252,7 @@ fn test_garbage_collection() {
     run_actix(async {
         let block_prod_time = 100;
         let epoch_length = 5;
-        let target_height = epoch_length * (NUM_EPOCHS_TO_KEEP_STORE_DATA + 1);
+        let target_height = epoch_length * (MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA + 1);
```

Comment:
In tests we need to use the default value which probably should remain 5. We might want two constants ‘DEFAULT_NUM_EPOCHS_…’ and ‘MIN_NUM_EPOCHS_…’ or maybe use ‘GCConfig::default().num_epochs_to_keep()’ as the default.

Reply:
Changed it to `DEFAULT_NUM_EPOCHS_...`